site/cgi-bin/pdf_check.cgi

```diff
@@ -83,8 +83,8 @@ try:
         for j in range(0,div):
             out_pdf = filename[:-4] + "_" + str(j) + ".pdf"
             out_cover_pdf = filename[:-4] + "_" + str(j) + "_cover.pdf"
-            subprocess.call("pdftk {in_pdf} cat {start}-{stop} output {out_pdf}".format(in_pdf=filename,start=j*num+1,stop=(j+1)*num,out_pdf=out_pdf),shell=True)
-            subprocess.call("pdftk {in_pdf} cat {start} output {out_pdf}".format(in_pdf=filename,start=j*num+1,out_pdf=out_cover_pdf),shell=True)
+            subprocess.call("pdftk \"{in_pdf}\" cat {start}-{stop} output \"{out_pdf}\"".format(in_pdf=filename,start=j*num+1,stop=(j+1)*num,out_pdf=out_pdf),shell=True)
+            subprocess.call("pdftk \"{in_pdf}\" cat {start} output \"{out_pdf}\"".format(in_pdf=filename,start=j*num+1,out_pdf=out_cover_pdf),shell=True)
```

Comment:
Don't use shell=True in `subprocess` calls unless you absolutely need to. Use `subprocess.call(['pdftk', filename, 'cat', ...])`. Apply above to `subprocess.check_output`